libswirl/gui/gui.cpp

```diff
@@ -354,6 +354,8 @@ struct ReicastUI_impl : GUI {
             std::string game_file = cfgLoadStr("config", "image", "");
             if (!game_file.empty())
             {
+                if(game_file == "bios")
```

Comment:
Do you mind adjusting the logic in `libswirl.cpp::int GetFile(char* szFileName)`? You'll have to change a few empty strings as well in GUI, but it's a much cleaner solution to make "bios" mean "bios" at the core level. I never liked using an empty string anyway.

Reply:
Hmm, also maybe name it no "nodisk" to make it clear what it does?